src/Concordium/Client/Output.hs

```diff
@@ -195,20 +196,38 @@ showRevealedAttributes as =
         Just k -> Text.unpack k
     showAttr (t, IDTypes.AttributeValue v) = printf "%s=%s" (showTag t) (show v)
 
+prettyPrintTokens :: [Types.Token] -> [String]
+prettyPrintTokens = map formatToken
+  where
+    indent = replicate 24 ' '
+    formatToken (Types.Token tid (Types.TokenAccountState (Types.TokenAmount digits decs) inAllowList inDenyList)) =
+        let amount = fromIntegral digits / (10 ^ decs :: Double)
+        in  unlines
+                [ indent ++ "Token ID:            " ++ show tid,
+                  indent ++ "Balance:             " ++ printf ("%." ++ show decs ++ "f") amount,
+                  indent ++ "In Allow List:       " ++ show inAllowList,
+                  indent ++ "In Deny List:        " ++ show inDenyList
+                ]
```

Comment:
I would suggest just having one line per PLT and having it something like
```
    50.000000 PLT
    0.100000  ETH
```
I would leave out the allow and deny list for now, as even the interface for them is subject to change.

Reply:
I can make it one line but would like to keep the info about the `allow` and `deny` list since it is valid info.

Reply:
![Screenshot from 2025-05-08 12-55-42](https://github.com/user-attachments/assets/702777b3-1563-4e70-8a00-2e7c637a7516)
To remove the quotes around "ETH" we can implement some custom show instead of deriving it at the types in another PR.
https://github.com/Concordium/concordium-base/blob/plt/haskell-src/Concordium/Types/Tokens.hs#L19 (edited)